canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -58,6 +58,13 @@ <h4 class="list-group-item-heading">Find Course Info</h4>
           </a>
         </div>
 
+        <div class="list-group col-md-4">
+          <a type="button" class="btn btn-primary list-group-item active" href="{% url 'cross_list_courses:index' %}" class="list-group-item active">
+              <h4 class="list-group-item-heading">Cross-list Courses</h4>
+              <p class="list-group-item-text">Set up and modify cross-listings <br>&nbsp;</p>
```

Comment:
@Vittorio2015 I would hesitate to bring in another framework just to make this more CSS-like. There are custom solutions, using flex-box for example, but if what we have works, let's stick with it for now! By all means look into it if you're curious and have time though, but what we have works and isn't egregious or anything. :)